proto/ibc/core/client/v1/query.proto

```diff
@@ -148,14 +155,13 @@ message QueryClientParamsResponse {
   // params defines the parameters of the module.
   Params params = 1;
 }
+
 // QueryUpgradedClientStateRequest is the request type for the
 // Query/UpgradedClientState RPC method
 message QueryUpgradedClientStateRequest {
-  // client state unique identifier
-  string client_id = 1;
-  // plan height of the current chain must be sent in request
-  // as this is the height under which upgraded client state is stored
-  int64 plan_height = 2;
+  // client id and plan height field have been deprecated. Use gRPC header for setting the height instead.
+  reserved 1, 2;
+  reserved "client_id", "plan_height";
```

Comment:
I think we can just remove these. I don't think we have done a release with this proto definition (since it was added when removing IBC from x/upgrade)